src/Config/CliBuilder.ts

```diff
@@ -41,11 +41,19 @@ export class CliBuilder {
           'What level of logs to report. Any logs of a higher level than the setting are shown.'
       })
       .option('api', {
-        default: 'https://nexploit.app/',
         requiresArg: true,
         demandOption: true,
         describe: 'NexPloit base URL'
       })
+      .option('bus', {
+        requiresArg: true,
+        demandOption: true,
+        describe: 'NexPloit Event Bus'
+      })
```

Comment:
Please mark them as deprecated, and extend the description by adding a default value or example